Doc/library/shutil.rst

```diff
@@ -39,16 +39,19 @@ Directory and files operations
 
 .. function:: copyfileobj(fsrc, fdst[, length])
 
-   Copy the contents of the file-like object *fsrc* to the file-like object *fdst*.
-   The integer *length*, if given, is the buffer size. In particular, a negative
-   *length* value means to copy the data without looping over the source data in
-   chunks; by default the data is read in chunks to avoid uncontrolled memory
-   consumption. Note that if the current file position of the *fsrc* object is not
-   0, only the contents from the current file position to the end of the file will
-   be copied.
+   Copy the contents of the file-like object *fsrc* to the file-like object
+   *fdst*.
 
+   Note that if the current file position of the *fsrc* object is not 0,
+   only the contents from the current file position to the end of the file
+   will be copied.
```

Comment:
IMO “Note that” is redundant. In fact the whole first line could be dropped, and “Only the contents from the current file position . . .” could be put back in the first paragraph.

Reply:
I agree with this.

Reply:
Done!